backend/src/main/java/de/kathahuth/gymast/model/Group.java

```diff
@@ -0,0 +1,17 @@
+package de.kathahuth.gymast.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Document(collection = "groups")
```

Comment:
Maybe add @Builder and use the builder methods for your tests?!
This will help you save time later in the project in the case you have to expand your Group-model.